ansible/roles/server-swap/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for roles/server-swap
+
+sysctl_vm_swappiness: 1
```

Comment:
Kernel version 3.5 and over, as well as kernel version 2.6.32-303 and over: Minimum amount of swapping without disabling it entirely.